lib/travis/build/bash/travis_cmd.bash

```diff
@@ -54,9 +54,9 @@ travis_cmd() {
     fi
     result="${?}"
     if [[ -n "${secure}" && "${result}" -ne 0 ]]; then
-      echo -e "${ANSI_RED}The previous command failed, possibly due to a malformed secure environment variable.${ANSI_CLEAR}
-${ANSI_RED}Please be sure to escape special characters such as ' ' and '$'.${ANSI_CLEAR}
-${ANSI_RED}For more information, see https://docs.travis-ci.com/user/encryption-keys.${ANSI_CLEAR}"
+      echo -e "${ANSI_RED}We were unable to parse one of your secure environment variables.${ANSI_CLEAR}
+${ANSI_RED}Please make sure to escape special characters such as ' ' and $ with .${ANSI_CLEAR}
```

Comment:
Is there something missing after "with "?

Reply:
"with ." (dot sign). This it how it was described in task.
Maybe it should be explained more precisely with "(dot sign)" description after . sign.
What do you think @MariadeAnton ?

Reply:
Perhaps we can be more explicit in each of these cases, so that the text printed on the logs is:

    Please make sure to escape special characters such as ' ' (white space) and $ (dollar symbol) with \ (backslash) .

Reply:
that'd be super helpful, yes